### PR TITLE
Landing page touches

### PR DESCRIPTION
**Pull request by dawkaka**
Screenshot from 2023-05-24 00-21-53
Screenshot from 2023-05-24 00-21-35

### CODE REVIEW

**Comment by tonyalaribe** [comment]
Hey man, thanks a lot of the changes. I'm reviewing it and quite like some of the changes, like the glassy nav bar. Though some of the changes not as much. But I'll try to merge the components I like into master on a separate branch. Is that ok?

File: layouts/docs/content.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+<section class="flex flex-col md:flex-row divide-x divide-slate-200 border-y border-slate-200" style="top: 10px;">
   <aside class="md:w-1/5 text-slate-800 text-md px-2">
     {{ partial "sidebar-tree.html" . }}
   </aside>
   <main class="grow p-5 lg:p-8">
     <div class="td-content prose  prose-slate py-5">
       <h1>{{ .Title }}</h1>
       {{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
-      <header class="article-meta">
+      <header clasks="article-meta">
```
Comment (tonyalaribe) [inline layouts/docs/content.html:9]: typo?

**Comment by dawkaka** [comment]
yes

**Comment by dawkaka** [comment]
Yeah, corrected.

**Pull request closed without merging**